builds.yml

```diff
@@ -189,6 +189,7 @@ env:
   - DEBUG: null
   - SUPPORT_LINK: https://support.metamask.io
   - SUPPORT_REQUEST_LINK: https://metamask.zendesk.com/hc/en-us
+  - METAMETRICS_SETTINGS_LINK: https://support.metamask.io/privacy-and-security/how-to-manage-your-metametrics-settings
```

Comment:
Does this need to be a built time env var?  Why not just a javascript constant?

Reply:
Let's embed this link @jonybur ; the reason some of these are in `builds.yml` is because different build types (BETA, Flask) have different links for those items; this link will be static so we can just embed it.